global/scripts/RSNregression.m

```diff
@@ -355,7 +348,7 @@ function cleanupFunc(tempbase)
 end
 
 %like call_fsl, but without sourcing fslconf
-function my_system(command)
+function [exitStatus, stdout]=my_system(command)
```

Comment:
Since this function already errors if the exit status is nonzero, it doesn't need to return the exit status.

Reply:
I think it might be a good idea to have the same output structure as `system`. Don't you think so?

Reply:
No.  `my_system` is intended to do the error detection that most people forget to do when they use `system`.  If `my_system` returns, the exit status was 0, so there is no reason to return it, so the only time you should use its return value is when you want the stdout.  It is not a drop-in replacement or wrapper, it is a simplification.

I might also argue that it should only use `'-echo'` when the nargout is 0.